src/Microsoft.TemplateEngine.Orchestrator.RunnableProjects/ConfigModel/TemplateConfigModel.cs

```diff
@@ -434,6 +434,11 @@ internal static TemplateConfigModel FromJObject(JObject source, ILogger? logger
             return new TemplateConfigModel(source, logger, baselineName, filename);
         }
 
+        internal void RemoveSymbol(string name)
```

Comment:
weirdo: should probably consider the weird case when the user defines `name` parameter and it's disabled, and return to implicit symbol here :)